core-services/prow/02_config/generate-boskos.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import yaml
+
+
+CONFIG = {
+    'aws-quota-slice': {
+        # Wild guesses.  We'll see when we hit quota issues
+        'us-east-1': 50,
+        'us-east-2': 50,
+        'us-west-1': 50,
+        'us-west-2': 50,
+    },
+    'azure4-quota-slice': {
+        # Cannot create more than 50 public IP addresses for this subscription in this region.
+        # and each cluster needs three of public IPs: https://docs.openshift.com/container-platform/4.5/installing/installing_azure/installing-azure-account.html#installation-azure-limits_installing-azure-account
+        'centralus': 16,
+        # Wild guesses.  We'll see when we hit quota issues
+        'eastus1': 10,
+        'eastus2': 10,
+        'westus': 10
+    },
+    'gcp-quota-slice': {
+        'us-east1': 120,
+    },
+    'libvirt-s390x-quota-slice': {},
+    'libvirt-ppc64le-quota-slice': {},
+    'metal-quota-slice': {
+        # Wild guesses.  We'll see when we hit quota issues
+        'default': 1000,
+    },
+    'openstack-quota-slice': {
+        'default': 7,
+    },
+    'openstack-vexxhost-quota-slice': {
+        'default': 2,
+    },
+    'openstack-ppc64le-quota-slice': {},
+    'ovirt-quota-slice': {},
+    'packet-quota-slice': {
+        'default': 10,
+    },
+    'vsphere-quota-slice': {
+        # VMC is unable to handle the thundering herd problem.
+        'default': 10,
+    },
+}
+
+for i in range(2):
+    for j in range(5):
+        CONFIG['libvirt-s390x-quota-slice']['libvirt-s390x-{}-{}'.format(i, j)] = 1
+
+for i in range(2):
+    for j in range(4):
+        CONFIG['libvirt-ppc64le-quota-slice']['libvirt-ppc64le-{}-{}'.format(i, j)] = 1
+
+for i in range(4):
+    CONFIG['openstack-ppc64le-quota-slice']['openstack-ppc64le-0{}'.format(i)] = 1
+
+for i in range(8):
+    CONFIG['ovirt-quota-slice']['ovirt-1{}'.format(i)] = 1
```

Comment:
similar?

Reply:
Pivoted to `range(10, 18)` with 204dc967ab -> b831dd7a56.